helpdesk_mgmt/data/helpdesk_data.xml

```diff
@@ -102,13 +102,15 @@
             <field name="name">In Progress</field>
             <field name="unattended">False</field>
             <field name="closed">False</field>
+            <field name="mail_template_id" ref="helpdesk_mgmt.changed_stage_template" />
```

Comment:
Don't add this by default. Customize it in your end.

Reply:
The `changed_stage_template` exists but is not used anywhere. For me it is more logical to add it by default and so easier customize it by simply removing if from stages.

Reply:
No, it's not a good default. People not aware of this will have mails sent on any move. It's there, but not filled, because it's optional.